SequenceBrowser/Widgets/qMRMLSequenceBrowserSeekWidget.cxx

```diff
@@ -179,7 +179,7 @@ void qMRMLSequenceBrowserSeekWidget::updateWidgetFromMRML()
     d->label_IndexValue->setText(indexValue);
     d->label_IndexUnit->setText(indexUnit);
 
-    if (!indexValue.length() == 0)
+    if (indexValue.length() != 0)
```

Comment:
@Sunderlandkyl could you check this?

It would be also nice to add a comment describing why these few lines are needed.

Reply:
Seems fine to me.
I'll integrate this and add some documentation.